src/GuzzleBundleCachePlugin.php

```diff
@@ -45,6 +47,28 @@ public function loadForClient(array $config, ContainerBuilder $container, string
             $cacheMiddlewareExpression = new Expression(sprintf('service("%s")', $cacheMiddlewareDefinitionName));
 
             $handler->addMethodCall('push', [$cacheMiddlewareExpression, 'cache']);
+
+            $eventDispatcherName       = sprintf('guzzle_cache.event_dispatcher.%s', $clientName);
```

Comment:
better let's use `guzzle_bundle_cache_plugin.event_dispatcher.%s` to keep consistency with middleware definition

Reply:
agreed